### ISSUE TITLE
Observables not working as expected

### ISSUE THREAD

**Issue by brianmhunt** [cssinjs/jss#735]
Hi again :)

Observables aren't working as expected.  In particular, I've a TC39 implementation of Observables which is not being interpreted correctly.

The problem would seem to be the `isObservable` function, which adds this to `jss`:

```js
  var symbolObservable = lib;

  var isObservable = (function (value) {
    return value && value[symbolObservable] && value === value[symbolObservable]();
  });
```

Here's some command line output that's illuminating:

```js
>>> Symbol.observable === symbolObservable
false
>>> symbolObservable.default === Symbol.observable
true
```

It's possible there's some weird things going on with the build process on my end (perhaps I need some webpack-commonjs bridge), but I didn't see any tests in jss that demonstrate the use of `Symbol.observable` proper, so I can't easily eliminate the possibility that it's a jss issue.

**Comment by kof** [comment]
Can symbol observable package be a problem? So either there was a change and versions are incompatible (I doubt that) or you evtl compiled 2 different versions and they have different refs.

In any case we need to address that, I hope you can come up with a change or a PR for this.

**Comment by kof** [comment]
> symbolObservable.default === Symbol.observable

this sounds like you got a problem with a mix of esm and cjs.

**Comment by brianmhunt** [comment]
Thanks.  The build-process is as vanilla a Webpack 4 config as one could imagine, so it should do CommonJs as expected, but evidently ... something's awry.

Incidentally, it would be quite a bit easier to diagnose if there were a base jsFiddle or CodePen.

**Comment by brianmhunt** [comment]
FWIW, this looks like a build issue; but a weird one.  For interest, the webpack output is 

```js
  unwrapExports(lib);

  var symbolObservable = lib;
```

But it should be:
```js
  var symbolObservable = unwrapExports(lib);
```

Will report as I work through it.

**Comment by brianmhunt** [comment]
Relatedly, when my code has this:

```
>>> import $$obs from 'symbol-observable'
>>> console.log("OBS", $$obs)
OBS Symbol(observable)
```

i.e. the import is only an issue when it's via the jss library.

**Comment by kof** [comment]
so you are using a dist version?

**Comment by brianmhunt** [comment]
Both are exhibiting the issue.

**Comment by brianmhunt** [comment]
i.e. both of these fail:
```
import jss from 'jss'
import jss from 'jss/dist/jss'
```

**Comment by brianmhunt** [comment]
... going to check a CDN version now.

**Comment by kof** [comment]
I guess it will be a problem with dist version if youb have a separate bundle and jss bundle has the symble built in, so those 2 bundles end up using different symbols.

**Comment by kof** [comment]
on a lib version, the should be using the same symbol.

**Comment by brianmhunt** [comment]
The symbol-observable polyfill will always work on a browser where `Symbol.observable` is defined, since that's what it returns.

Incidentally, unless I'm missing something really obvious, jss is failing from CDN:

```html
<!DOCTYPE html>
<html>
  <head>
    <script src='https://cdnjs.cloudflare.com/ajax/libs/jss/9.8.6/jss.js'></script>
  </head>
  <body>
  </body>
</html>
```

Console outputs:

>Uncaught ReferenceError: global is not defined  at escape.js:1     at _typeof (jss.js:4)     at jss.js:5
> (anonymous) @ escape.js:1
> _typeof @ jss.js:4
> (anonymous) @ jss.js:5

Which is the following line from `escape.css`:

> const CSS = (global.CSS: any)

So I can't seem to test with the CDN either.

**Comment by kof** [comment]
oh, migration to rollup broke that, webpack has a built-in plugin for "global" replacement

**Comment by brianmhunt** [comment]
Yes, these seem like the types of issues one gets with Rollup. :)  We had similar problems with tko/knockout.  For interest, our [rollup config](https://github.com/knockout/tko/blob/master/rollup.config.js)

**Comment by kof** [comment]
looks like we need to add https://www.npmjs.com/package/rollup-plugin-node-globals

**Comment by kof** [comment]
This won't fix the prob with symbols though

**Comment by kof** [comment]
You can try in the meantime an older version, which used webpack

**Comment by brianmhunt** [comment]
Thanks @kof; what's a recent version compiled with webpack?

**Comment by kof** [comment]
https://github.com/cssinjs/jss/blob/master/changelog.md#981--2018-03-19

**Comment by keeth** [comment]
Has anyone had any luck with this?  Just ran into it 3 years later.
